react/features/overlay/components/PageReloadOverlay.js

```diff
@@ -0,0 +1,298 @@
+/* global APP, AJS */
+
+import React, { Component } from 'react';
+
+import { randomInt } from '../../base/util/randomUtil';
+
+import AbstractOverlay from './AbstractOverlay';
+
+const logger = require('jitsi-meet-logger').getLogger(__filename);
+
+/**
+ * Implements a React Component for the reload timer. Starts counter from
+ * props.start, adds props.step to the current value on every props.interval
+ * seconds until the current value reaches props.end. Also displays progress
+ * bar.
+ */
+class ReloadTimer extends Component {
```

Comment:
I was trying to reduce the number of files because `ReloadTimer` is used only here. But I accept your arguments. I don't mind if we put it in a new file.